py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -494,12 +494,12 @@ def _check_header_consistency(_key, _test_val, atol, _consistent, _err_msg):
         self.get_var_lss = interp1d(log_lambda,
                                     var_lss,
                                     fill_value='extrapolate',
-                                    kind='nearest')
+                                    kind='cubic')
 
         self.get_mean_flux = interp1d(log_lambda,
                                       mean_flux,
                                       fill_value='extrapolate',
-                                      kind='nearest')
+                                      kind='cubic')
```

Comment:
I have a warning here. Depending on the input file and its noise levels, cubic spline can act terribly. I remember the files from rawio analysis being very noisy. Andrei, you created a smooth spline version. Has that become the norm?

Reply:
Good point. Yes the new raw-stats files also save a spline version, but I don't think it's the default. The current files in picca (for 2.4A and 3.2A pixelization) are old and don't have that. So I'm gonna look into updating those as well.